main.go

```diff
@@ -34,7 +34,7 @@ func main() {
 	c := passenger.Context{}
 	factory := passenger.CreateFactory(c)
 	server := factory.FindInstance()
-	for i := 0; i < 20; i++ {
+	for i := 0; i < 2000; i++ {
```

Comment:
Thanks for requesting.
How about like this?

```suggestion
var (
  timeout = flag.Int("timeout", 30, "Maximum time (in seconds) to wait for passenger instance.")
)

func main() {
  maxRetries := (*timeout * 1000) / 200
  for i :=0; i < maxRetries; i++ {
  ...
  }
}
```

Reply:
Hi @kyanagimoto 
I just updated the PR
please check again